webapp/components/survey/LabelsEditor/Row/Row.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import * as R from 'ramda'
+
+import { Input } from '@webapp/components/form/input'
+
+import Badge from '../Badge'
```

Comment:
if Badge is only used here, the components should be added here not to parent folder